mappings/net/minecraft/entity/ai/AIGoals.mapping

```diff
@@ -1,17 +1,17 @@
-CLASS net/minecraft/class_1790 net/minecraft/entity/ai/AIGoals
+CLASS net/minecraft/class_1790 net/minecraft/entity/ai/AiGoals
 	FIELD field_7650 LOGGER Lorg/apache/logging/log4j/Logger;
 	FIELD field_7652 goalEntries Ljava/util/List;
 	FIELD field_7653 profiler Lnet/minecraft/class_1705;
 	METHOD <init> (Lnet/minecraft/class_1705;)V
 		ARG 1 profiler
-	METHOD method_7371 addGoal (ILnet/minecraft/class_1789;)V
```

Comment:
addGoal is a better name

Reply:
![image](https://user-images.githubusercontent.com/62552372/96240656-ccda6600-0fbe-11eb-990d-258101e92daf.png)
the class only adds one thing

Reply:
upsream calls it `add`